### PR TITLE
Create the base for our networking layer

### PR DESCRIPTION
**Pull request by tikipatel**

### CODE REVIEW

File: Earthquakes/Earthquakes.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -24,6 +26,8 @@
 		8E6535DD1DC018150078516A /* Base */ = {isa = PBXFileReference; lastKnownFileType = file.storyboard; name = Base; path = Base.lproj/LaunchScreen.storyboard; sourceTree = "<group>"; };
 		8E6535DF1DC018150078516A /* Info.plist */ = {isa = PBXFileReference; lastKnownFileType = text.plist.xml; path = Info.plist; sourceTree = "<group>"; };
 		8E6535E51DC01D710078516A /* Earthquake.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = Earthquake.swift; sourceTree = "<group>"; };
+		8E6535E71DC0348C0078516A /* NetworkManager.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = NetworkManager.swift; sourceTree = "<group>"; };
```
Comment (RRiOSDev) [inline Earthquakes/Earthquakes.xcodeproj/project.pbxproj:29]: Love this entry 👍

File: Earthquakes/Earthquakes.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -13,6 +13,8 @@
 		8E6535DB1DC018150078516A /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 8E6535DA1DC018150078516A /* Assets.xcassets */; };
 		8E6535DE1DC018150078516A /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 8E6535DC1DC018150078516A /* LaunchScreen.storyboard */; };
 		8E6535E61DC01D710078516A /* Earthquake.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8E6535E51DC01D710078516A /* Earthquake.swift */; };
```
Comment (patrickgarner) [inline Earthquakes/Earthquakes.xcodeproj/project.pbxproj:15]: Pat's comment!

**Comment by madisonquanisha** [comment]
Great code dude!!!

File: Earthquakes/Earthquakes.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -13,6 +13,8 @@
...
+		8E6535E81DC0348C0078516A /* NetworkManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8E6535E71DC0348C0078516A /* NetworkManager.swift */; };
+		8E6535EA1DC035090078516A /* NetworkManager+Enumerations.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8E6535E91DC035090078516A /* NetworkManager+Enumerations.swift */; };
 /* End PBXBuildFile section */
```
Comment (nalula1) [inline Earthquakes/Earthquakes.xcodeproj/project.pbxproj:19]: //break time

File: Earthquakes/Earthquakes.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -13,6 +13,8 @@
 		8E6535DB1DC018150078516A /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 8E6535DA1DC018150078516A /* Assets.xcassets */; };
 		8E6535DE1DC018150078516A /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 8E6535DC1DC018150078516A /* LaunchScreen.storyboard */; };
 		8E6535E61DC01D710078516A /* Earthquake.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8E6535E51DC01D710078516A /* Earthquake.swift */; };
+		8E6535E81DC0348C0078516A /* NetworkManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = 8E6535E71DC0348C0078516A /* NetworkManager.swift */; };
```
Comment (RRiOSDev) [inline Earthquakes/Earthquakes.xcodeproj/project.pbxproj:16]: This is some new shit I coded

File: Earthquakes/Earthquakes.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -13,6 +13,8 @@
 		8E6535DB1DC018150078516A /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 8E6535DA1DC018150078516A /* Assets.xcassets */; };
 		8E6535DE1DC018150078516A /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 8E6535DC1DC018150078516A /* LaunchScreen.storyboard */; };
```
Comment (patrickgarner) [inline Earthquakes/Earthquakes.xcodeproj/project.pbxproj:14]: who the fuck put uninsulated tiles on the shuttle (#12163)

**Review by RRiOSDev — APPROVED** [review]

**Review by delkant — APPROVED** [review]
Good Job!!

File: Earthquakes/Earthquakes.xcodeproj/project.pbxproj (hunk 5 above)
Comment (nalula1) [reply Earthquakes/Earthquakes.xcodeproj/project.pbxproj:14]: it was me! tony turnip

**Review by patrickgarner — APPROVED** [review]
Holy hell we got some of the battle queue going.

**Pull request merged**